library/Fission/IPFS/Peer.hs

```diff
@@ -3,14 +3,21 @@ module Fission.IPFS.Peer
   , rawList
   , connect
   , fission
+  , getExternalAddress
   ) where
 
 import           RIO hiding (all)
 import qualified RIO.ByteString.Lazy as Lazy
 import qualified RIO.Text            as Text
 import           RIO.Process (HasProcessContext)
 
+import           Net.IPv4
+import           Text.Regex
+
 import           Data.Has
+import           Data.List hiding (all)
+import qualified Data.Text as T
```

Comment:
Protip: you have these functions already exported from `RIO.Text` (which I believe just re-exports `Data.Text`). As a general pattern, I tend to recommend using `Text` instead of `T` for clarity sake.